scholarx/2023/index.html

```diff
@@ -48,22 +48,18 @@
   <section class="section pb-0 bg-gradient-white mt-5">
     <div class="container">
       <div class="row row-grid align-items-center justify-content-center">
-        <div class="col-md-6">
-          <div class="d-flex px-3">
-            <div class="pl-4">
+        <div class="col-md-6 px-3 pl-4 mb-5">
               <img src="/assets/img/brand/scholarx.png" class="img-fluid align-self-center mb-5 w-50">
-              <p>ScholarX is an exclusive 6-month program aimed at providing
-                mentoring support to a selected pool of high Potential undergraduate students
-                based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
-                the world’s top universities or Fortune 500 companies. It’s our free premium
-                mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
-                of knowledge and expertise sharing without the limitation of geographical borders.</p>
+              <p class="pr-1 mt-0">ScholarX intends to deliver free premium mentoring assistance for the elite undergraduate
+                students based in Sri Lanka, through the means of a Sri Lankan expatriate presently involved
+                with one of the world’s most renowned universities or Fortune 500 companies.
+                This program strives to create a sustainable education structure within Sri Lanka by
+                drawing knowledge and expertise from all across the globe.
+              </p>
             </div>
 
-          </div>
-        </div>
-        <div class="col-md-6">
-          <img src="/assets/img/scholarx/new-findmentor.png" class="img-fluid align-self-center w-100">
+        <div class="col-md-6 align-self-center my-0">
+          <img src="./images/mentor-mentee archive/mentor-mentee archive image.png" alt="mentor-mentee archive image" class="img-fluid align-self-end w-100 px-5 py-0">
```

Comment:
Let rename the image to sx-hero-image @MethmaPeiris99

Reply:
I'll rename it